onnxruntime_extensions/tools/add_pre_post_processing_to_transformers.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import shutil
+import re
+import tempfile
+import functools
+
+from pathlib import Path
+
+from pre_post_processing import PrePostProcessor, Debug
+from pre_post_processing.steps import *
+from pre_post_processing.utils import create_named_value
+
+# for results verification
+import transformers
+import torch
+import onnxruntime
+import numpy as np
+import onnxruntime_extensions
+
+
+# avoid loading model from huggingface multiple times, it's time consuming
+@functools.lru_cache
+def get_tokenizer_and_model_from_huggingface(model_name):
```

Comment:
Can we separate things out so that this is purely a script to add the pre/post processing to a model (or update the existing script with that specific code?

Parts to acquire a model and validate could go in tutorials as that's similar to \tutorials\superresolution_e2e.py.

Reply:
I  believe seperating this out is a good idea.  But I got a quetion, How could users provide tokenizers and pass to the tool if there is only pure pre-post function?

Reply:
I can imagine you might have some helpers in the script that uses the Step that know about a specific tokenizer and can extract the necessary info from it. Possibly just a case of moving that logic up a level instead of doing it inside the Step.

Reply:
Have seperated out into tool and tutorial for bert_e2e models